metalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Linters": {
+	      "errcheck": "errcheck -abspath -ignore 'Close|Write|Serve' {path}:PATH:LINE:COL:MESSAGE",
+	      "license": "grep -rL 'Apache License' {path}:^(?P<path>.*?\\.go.*)"
+             },
+  "Severity": {"license": "error"},
+  "Enable": ["gofmt", "golint", "vet", "ineffassign", "misspell", "gocyclo"],
+  "Vendor": true,
+  "VendoredLinters": true,
```

Comment:
- `Vendor` - means "this project is using vendoring" - ignore packages in the vendor folder.
- `VendoredLinters` means use the linters that were vendored by `gometalinter`. This is a bit of insurance against upstream changes in the linters themselves.